inputs-required.tf

```diff
@@ -28,6 +28,10 @@ variable "product" {
   type        = string
 }
 
+variable "project" {
```

Comment:
README changes are missing in example and values table which shows that there is missing CI enforcing README updates.

https://github.com/hmcts/terraform-module-servicebus-namespace/blob/master/.github/workflows/terraform-docs.yml